src/guides/v2.3/test/svc/semver_test_execution.md

```diff
@@ -0,0 +1,37 @@
+---
+group: testing
+title: Running Semantic Version Checker
+functional_areas:
+  - Testing
+  - test
+---
+
+Executing the Magento 2 Semantic Version Checker is a simple process, but it requires an external tool.
+
+### Running Semantic Version Checker on local changes
+
+To run the Semantic Version Checker on local changes:
+
+1. Clone the `magento-semver` tool to your local machine:
+
+   ```bash
+   git clone git@github.com:magento/magento-semver.git
+   ```
+
+1. Navigate to the cloned repository and run `composer install` to install the project dependencies:
+
+   ```bash
+   cd magento-semver && composer install
+   ```
+
+1. To run the Semantic Version Checker, we need one folder with the feature changes and another folder with mainline code (without changes),
+so you may need to clone the Magento repository to a separate folder to perform the comparison.
+
+1. Navigate the `magento-semver` folder and run the Semantic Version Checker compare command:
+
+   ```bash
+   bin/svc compare ../magento2-mainline ../magento2
+   ```
+
+The first parameter is the mainline code without any changes, and the second parameter is the path to the folder with your changes.
+This command you output the results of the Semantic Version Checker tool.
```

Comment:
Where do the results output to? Is it obvious?

Reply:
The output is the result of the CLI command in this case, I guess there is a typo there, it should be:
```This command will output the results of the Semantic Version Checker tool.```

Maybe we can make it more explicit by saying:
```This command will output on the console the results of the Semantic Version Checker tool.```

What do you think @dobooth ?

Reply:
The results of the Semantic Version Checker are outputted to the console.